web/src/components/job/JobList.js

```diff
@@ -21,6 +22,14 @@ const JobList = () => {
   const [rowData, setRowData] = useState([]);
   const [redirect, setRedirect] = useState();
   const [deleteJobId, setDeleteJobId] = useState();
+  const iconViewBoxDimension = '-2 -2 30 30';
```

Comment:
Try to avoid using transforms, especially for elements in grids where there are many rows.